pints/_log_likelihoods.py

```diff
@@ -117,6 +117,120 @@ def __call__(self, x):
                       - np.sum(autocorr_error**2, axis=0) / (2 * sigma**2))
 
 
+class GaussianIntegratedUniformLogLikelihood(pints.ProblemLogLikelihood):
+    """
+    Calculates a log-likelihood assuming independent Gaussian-distributed noise
+    at each time point where it is assumed that the dependence on
```

Comment:
What do you mean when you say "it is assumed that the dependence on ... has been integrated out"? Is this something the user should have done before using this method?

Reply:
@MichaelClerx  You're right -- I'm not explaining this well currently! This method essentially removes sigma from the estimation problem so, for example, for the logistic problem, we end up only needing to do inference on the two parameters of the model. I've added additional information to the example notebook and the docstrings to hopefully explain this better.

Reply:
Sorry I really don't like this wording. Surely if you can do this that means you have integrated something, rather than you're assuming it has been done?